### PR TITLE
Fix links and code examples in addons/knobs

### PR DESCRIPTION
**Pull request by SilencerWeb**
Issue: broken links and not appropriate code examples to screenshot

## What I did
Fixed links and code examples

### CODE REVIEW

File: addons/knobs/README.md [hunk 1]
```diff
@@ -16,9 +16,9 @@ You can also use Knobs as a dynamic variable inside stories in [Storybook](https
 
 This is how Knobs look like:
 
-[![Storybook Knobs Demo](docs/storybook-knobs-example.png)](https://git.io/vXdhZ)
+[![Storybook Knobs Demo](docs/storybook-knobs-example.png)](https://goo.gl/uX9WLf)
 
-> Checkout the above [Live Storybook](https://storybooks-official.netlify.com/) or [watch this video](https://www.youtube.com/watch?v=kopW6vzs9dg&feature=youtu.be).
+> Checkout the above [Live Storybook](https://goo.gl/uX9WLf) or [watch this video](https://www.youtube.com/watch?v=kopW6vzs9dg&feature=youtu.be).
```
Comment (Hypnosphi) [inline addons/knobs/README.md]: This example uses some pretty outdated version of storybook. Please use this one instead:
https://storybooks-official.netlify.com/?knob-Dollars=12.5&knob-Name=Storyteller&knob-Years%20in%20NY=9&knob-background=%23ffff00&knob-Age=70&knob-Items%5B0%5D=Laptop&knob-Items%5B1%5D=Book&knob-Items%5B2%5D=Whiskey&knob-Other%20Fruit=lime&knob-Birthday=1484870400000&knob-Nice=true&knob-Styles=%7B%22border%22%3A%223px%20solid%20%23ff00ff%22%2C%22padding%22%3A%2210px%22%7D&knob-Fruit=apple&selectedKind=Addons%7CKnobs.withKnobs&selectedStory=tweaks%20static%20values&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs

**Review by Hypnosphi — DISMISSED** [review]
see comment

**Review by danielduan — APPROVED** [review]

**Comment by Hypnosphi** [comment]
I think we need to take new screenshots as well

**Review by Hypnosphi — CHANGES_REQUESTED** [review]
"Live Storybook with Knobs" link is still outdated:

https://github.com/SilencerWeb/storybook/blob/d1949e353c58934a357bc51b17e3a8705f3a1e64/addons/knobs/README.md#additional-links

**Comment by danielduan** [comment]
@Hypnosphi I removed that section because half the links don't work and the other half are duplicated from the top section

**Review by Hypnosphi — APPROVED** [review]

**Review by wuweiweiwu — APPROVED** [review]

**Pull request merged**